src/main/java/org/jabref/logic/texparser/DefaultTexParser.java

```diff
@@ -67,23 +66,29 @@ public TexParserResult parse(Path texFile) {
 
     @Override
     public TexParserResult parse(List<Path> texFiles) {
+        texParserResult.addFiles(texFiles);
         List<Path> referencedFiles = new ArrayList<>();
 
-        result.addFiles(texFiles);
-
         for (Path file : texFiles) {
-            try (LineNumberReader lnr = new LineNumberReader(Files.newBufferedReader(file))) {
-                for (String line = lnr.readLine(); line != null; line = lnr.readLine()) {
+            if (!file.toFile().exists()) {
+                LOGGER.error(String.format("File does not exist: %s", file));
+                continue;
+            }
+
+            try (LineNumberReader lineNumberReader = new LineNumberReader(Files.newBufferedReader(file))) {
+                for (String line = lineNumberReader.readLine(); line != null; line = lineNumberReader.readLine()) {
+                    // Skip comments and blank lines.
                     if (line.isEmpty() || line.charAt(0) == '%') {
```

Comment:
Here you should trim the line, because ` %` is also a commented out line

```java
if (line.trim().isEmpty() || line.trim().charAt(0) == '%') {
```